engine-comparison/report-gen/setOfCharts10.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```

Comment:
This file is almost identical to setOfCharts.html.  The javascript can be separated into a JS file that is then included in both HTML pages.

That said, I will probably be tinkering with the graphing quite a bit, so having you do that might be a waste of time.

Reply:
You're right, they are almost identical